packages/core/__tests__/Hub.test.ts

```diff
@@ -50,7 +54,9 @@ describe('Hub', () => {
 	});
 
 	test('Protected channel - ui', () => {
-		const listener = jest.fn(() => {});
+		const listener = jest.fn(() => {
+			// no-op
+		});
```

Comment:
Can we just drop the implementation if they're no-ops anyway?
```suggestion
		const listener = jest.fn();
```